app/panel/components/Summary.jsx

```diff
@@ -400,7 +400,7 @@ class Summary extends React.Component {
 						<div className={pageLoadClassNames}>
 							<span className="text">{t('page_load')} </span>
 							<span className="value">
-								{this.state.trackerLatencyTotal ? `${this.state.trackerLatencyTotal} ${t('settings_seconds')}` : '-'}
+								{this.state.trackerLatencyTotal ? `${this.state.trackerLatencyTotal} ${t('settings_seconds')}` : <React.Fragment>&mdash;</React.Fragment>}
```

Comment:
@zarembsky the dash is significantly larger than what it used to be, can we use another html entity that is more similar to what we have in production (`-`)?

Reply:
try `&ndash;`